### ISSUE TITLE
Discord channel

### ISSUE THREAD

**Issue by suciuale** [Teklad/csgo-horker#45]
why don't u make a discord channel?

**Comment by kazukeno** [comment]
Yes please make a discord + tell people that this cheat is undetected + updated often. Many people will benefit from it and valve is **rekt** because they cant do anything about it.

**Comment by suciuale** [comment]
@kazukeno i cannot tell if you're sarcastic rn lmao

**Comment by kazukeno** [comment]
Im not and it would be fun if they banned really much Windows cheaters like P2Cers and linux users with this cheat cant even get banned so that CS GO becomes easier and we feel cool for hacking when only private cheats work on Windows xD

**Comment by suciuale** [comment]
it was the same story with AimTux and it's forks lmao. give me your discord name and code to add you

**Comment by koutsie** [comment]
Lol if you really want a discord i can make one, just hmu in discord: Koutsie#0666

**Comment by ddjem87** [comment]
you know aimtux got detected cause was the most popular undetected cheat and vavle put alot of effort to stop this i really like this cheat cause is not very popular and its actually undetected so please everyone dont share it to much or make videos on it cause if it got some attention some vavle dev will notice and it will be the same story as aimtux (aimtux was internal tho so yea)

**Comment by Teklad** [comment]
AimTux was an internal hack which injects directly into the game's memory.  It got sigged really hard.  I don't think I've ever heard of a Linux external being detected to date.

**Comment by ddjem87** [comment]
you said they can search for procces names what should i change then cause i am not that experienced

please help us senpai.

**Comment by kazukeno** [comment]
I think that it can be randomized process names and then the cheat is safe lol. Teklad isn't that possible in c++?
